rust_src/remacs-modules/lib.rs

```diff
@@ -1,11 +1,14 @@
-extern crate remacs_sys;
 extern crate libc;
```

Comment:
Spacing and grouping matters. std, then external, then remacs, then local is the pattern the rest of the code follows.